subsys/net/ip/tcp2.c

```diff
@@ -192,7 +192,8 @@ static const char *tcp_th(struct net_pkt *pkt)
 	*s = '\0';
 
 	if (th->th_off < 5) {
-		s += snprintf(s, buf_size, "Bogus th_off: %hu", th->th_off);
+		s += snprintf(s, buf_size, "Bogus th_off: %hu",
```

Comment:
I wonder if we should use `snprintk()` as that is used more in networking code. Only caveat is that the snprintk() does not support all the modifiers.